src/components/vm/overview/memorySelectRow.jsx

```diff
@@ -37,59 +39,58 @@ class MemorySelectRow extends React.Component {
     }
 
     static getDerivedStateFromProps(nextProps, prevState) {
-        if (nextProps.value !== prevState.memory && !prevState.inputHasFocus)
+        if (nextProps.value != prevState.memory && !prevState.inputHasFocus)
```

Comment:
Out of interest, was that deliberate? Could this be a string vs. number comparison or so? Or accidental?

Reply:
Probably an accident, but I wrote this code a few months ago so I would like to recheck later today :) If it's accidental I will repost.

Reply:
Not a big deal either way, I was just curious. (Approval stands)

Reply:
Reposted with reverting this small change.